sam/imageutils.py

```diff
@@ -1546,24 +1544,14 @@ def write_dict_to_file(data_dict, to_file, bytes_per_voxel, header_offset):
     seek_time = 0
     write_time = 0
     seek_number = 0
-
     no_seek = 0
 
     for k in sorted(data_dict.keys()):
 
         seek_pos = int(header_offset + k * bytes_per_voxel)
-
-        if to_file.tell() != seek_pos:
-            # print "seek point:", seek_pos, to_file.tell()
-            seek_start = time()
-            to_file.seek(seek_pos, 0)
-            seek_number += 1
```

Comment:
i think this might still be relevant here, for benchmarking anyway, depending on if we define a seek to be a disk seek or just the os system call. @glatard thoughts?